tests/test_driver_info.py

```diff
@@ -0,0 +1,35 @@
+"""Test module for testing methods DriverInfo class from driver_info folder """
+import json
+from f1pystats.driver_info import DriverInfo
+
+
+class TestDriverInfo:
+    """Class containaing the functions for testing the DriverInfo class methods"""
+    
+    data = ""
+    with open("tests/test_data/first_3_drivers_info_2000.json", encoding='utf-8') as f:
+        data = json.load(f)
+        f.close()
+        
+    driver = DriverInfo(data)
+    
+    def test_driver_names(self):
+        """Test for the get_driver_names methods of DriverInfo"""
+        assert self.driver.get_driver_names() == ['Jean Alesi', 'Rubens Barrichello', 'Luciano Burti']
```

Comment:
Your test file calls ```get_driver_names()```, but there is no such function in ```DriverInfo```.

```DriverInfo``` contains a function called ```get_drivers_names()```. (NOTE the difference with ```drivers``` and ```driver```)